db/migrate/20220617173420_enable_follow_if_already_installed.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+class EnableFollowIfAlreadyInstalled < ActiveRecord::Migration[6.1]
+  def up
+    follow_installed_at = DB.query_single(<<~SQL)&.first
+      SELECT created_at FROM schema_migration_details WHERE version='20210901115010'
+    SQL
+
+    if follow_installed_at && follow_installed_at < 1.hour.ago
+      # The plugin was installed before we changed it to be disabled-by-default
+      # Therefore, if there is no existing database value, enable the plugin
```

Comment:
Actually, why would there have been no existing database value?

Reply:
We only store a value in the database if the user changes the value in the UI. Otherwise, we just pluck the default out of the yaml file.

Reply:
BTW, this was all stolen from  https://github.com/discourse/discourse-reactions/blob/main/db/migrate/20211022154420_enable_reactions_if_already_installed.rb

So in theory it is tried-and-tested 🤞